src/main/kotlin/no/nav/syfo/controllers/ReceivedSykmeldingController.kt

```diff
@@ -41,10 +44,45 @@ class ReceivedSykmeldingController(
         }
     }
 
+    suspend fun handlePapirsykmeldingFromSyfosmregister(
```

Comment:
Jeg skjønner ikke helt hvorfor denne er plassert her. Handler ikke denne klassen om mottak fra kafka (selv om den har et rart navn)? Og det er vel ikke syfosmregister som sender noe til smregistrering..?

Reply:
For at papirsykmeldinger som kommer fra syfosmregister skal kunne behandles i denne appen må de lagres i databasen - både for å ha historikk på endringer som gjøres, og for at det teknisk skal kunne passe inn med resten av flyten. Denne klassen tar egentlig bare inn en PapirSmRegistering og lagrer det til databasen, og brukes nå både ved mottak fra Kafka og ved henting fra syfosmregister.